contracts/cw1-subkeys/helpers.ts

```diff
@@ -231,7 +277,12 @@ const CW1 = (client: SigningCosmWasmClient): CW1Contract => {
       const result = await client.execute(contractAddress, {decrease_allowance: {spender, amount, expires}});
       return result.transactionHash;
     }
-    
+
+    const setupPermissions = async (spender: string, permissions: Permissions): Promise<string> => {
```

Comment:
Thanks for adding this here as well.

I think the `all_allowances` query is missing as well (my bad), can you add that too?